src/App.tsx

```diff
@@ -1,24 +1,157 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React, { useEffect, useState } from "react";
+import "antd/dist/antd.min.css";
+import "./App.css";
+import { Recipe } from "./recipe";
+import { Button, Collapse, Image } from "antd";
+import { DeleteOutlined, EditOutlined } from "@ant-design/icons";
 
 function App() {
+  const url = `${process.env.REACT_APP_URL}`;
+  if (url === undefined) {
```

Comment:
```suggestion
  if (!url) {
```
This is generally the syntax for what you're doing here.  It's technically more inclusive